data/importdata.sh

```diff
@@ -18,16 +18,40 @@ len=${#collections[@]}
 # Helpers
 ################################################################################
 create_database() {
-  az cosmosdb database create -g $resourceGroup -n $cosmosName --db-name $databaseName > /dev/null
+  echo "Checking whether the database '${databaseName}' already exists..."
+
+  dbExists="$(az cosmosdb database exists --db-name ${databaseName} -n ${cosmosName} --key ${password})"
+
+  if [[ "${dbExists}" = "false" ]]; then
```

Comment:
TIOLI: Can we add a user prompt, to see if they wanted to create a new DB/Collection? I'm thinking about the case where I have a collection named `Example`, but I accidentally put `Exampl`.

Reply:
+1 on above.

Reply:
Definitely could do. If that makes sense in the flow - ideally they are running this after running the `load_env.sh` script where these are stored in an vars.env file. Which makes more sense to do though, doing a 'just to be sure...do you want to make a new one?' or doing a 'make some edits to the vars.env file and re-`source` them'....? I could be convinced by either.

Reply:
I like scripts to ask all the questions at the beginning to avoid having to monitor it after execution.  You know long-running scripts provide a window to get coffee/tea.